bigquery_etl/cli/query.py

```diff
@@ -1186,6 +1195,55 @@ def initialize(name, sql_dir, project_id, dry_run):
                     job.result()
 
 
+@query.command(
+    help="""Render a query Jinja template.
+
+    Examples:
+
+    \b
+    # Backfill for specific date range
+    # second comment line
+    ./bqetl query render telemetry_derived.ssl_ratios_v1 \\
+      --output-dir=/tmp
```

Comment:
The comments don't seem to fit with this example.

Reply:
yikes